tutorials/LinearAlgebra/LinearAlgebra.ipynb

```diff
@@ -0,0 +1,986 @@
+    "    3 \\cdot 5 & 3 \\cdot 6 & 4 \\cdot 5 & 4 \\cdot 6 \\\\\n",
+    "    3 \\cdot 7 & 3 \\cdot 8 & 4 \\cdot 7 & 4 \\cdot 8\n",
+    "\\end{bmatrix}\n",
+    "=\n",
+    "\\begin{bmatrix}\n",
+    "    5 & 6 & 10 & 12 \\\\\n",
+    "    7 & 8 & 14 & 16 \\\\\n",
+    "    15 & 18 & 20 & 24 \\\\\n",
+    "    21 & 24 & 28 & 32\n",
+    "\\end{bmatrix}$$\n",
+    "\n",
+    "Notice that the tensor product of two vectors is another vector: if $V$ is an $n \\times 1$ vector, and $W$ is an $m \\times 1$ vector, $V \\otimes W$ is an $(n \\cdot m) \\times 1$ vector."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:blue\">Exercise 12</span>*: Tensor Product.\n",
+    "\n",
+    "**Inputs:**\n",
+    "\n",
+    "1. An $n \\times m$ matrix $A$.\n",
+    "2. A $k \\times l$ matrix $B$.\n",
+    "\n",
+    "**Goal:** Return $A \\otimes B$, the tensor product of $A$ and $B$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@exercise\n",
+    "def tensor_product(a : Matrix, b : Matrix) -> Matrix:\n",
+    "    return ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part III: Eigenvalues and Eigenvectors\n",
+    "\n",
+    "Certain quantum algorithms rely on the properties of so-called eigenvalues and eigenvectors. However, those concepts aren't necessary to start working with quantum computing. Feel free to move on to the [Qubit](../Qubit/Qubit.ipynb) and [Single Qubit Gates](../SingleQubitGates/SingleQubitGates.ipynb) tutorials, and come back here once you encounter eigenvalues and eigenvectors elsewhere.\n",
+    "\n",
+    "## Eigenvalues and Eigenvectors\n",
+    "\n",
+    "Consider the following example:\n",
+    "\n",
+    "$$\\begin{bmatrix}\n",
+    "    1 & -3 & 3 \\\\\n",
+    "    3 & -5 & 3 \\\\\n",
+    "    6 & -6 & 4\n",
+    "\\end{bmatrix}\n",
+    "\\begin{bmatrix}\n",
+    "    1 \\\\\n",
+    "    1 \\\\\n",
+    "    2\n",
+    "\\end{bmatrix}\n",
+    "=\n",
+    "\\begin{bmatrix}\n",
+    "    4 \\\\\n",
+    "    4 \\\\\n",
+    "    8\n",
+    "\\end{bmatrix}$$\n",
+    "\n",
+    "Notice that the resulting vector is just the initial vector multiplied by a scalar (in this case 4). There is a special name for these values in linear algebra. In general:\n",
+    "\n",
+    "Given a nonzero matrix $A$, nonzero vector $V$, and scalar $x$: if $AV = xV$, then $x$ is an **eigenvalue** of $A$, and $V$ is an **eigenvector** of that eigenvalue.\n",
+    "\n",
+    "Now, of course, this doesn't happen with every combination of matrix, vector, and scalar. Eigenvalues of a matrix are the roots of that matrix's **characteristic polynomial**. An $n \\times n$ matrix $A$'s characteristic polynomial is $|A - (x \\cdot I_n)| = 0$, where $|A|$ is, as you might remember, the matrix's determinant. \n",
```

Comment:
Actually, the more I think about this paragraph, the more I doubt that the learner needs it. We don't have a task on finding eigenvalues from scratch, so they won't use this information in the tutorial; we might as well keep it in the external resources. 

Alternatively, we can keep the text and add another exercise for finding eigenvalues of a 2x2 matrix (which ideally would be paired with an exercise for solving a quadratic equation in ComplexArithmetic tutorial). Or we can keep the tutorials as is for now and file an issue to add this pair of tasks later.

What do you think?